LPFilter/Source/PluginProcessor.cpp

```diff
@@ -167,15 +176,36 @@ void LpfilterAudioProcessor::processBlock (AudioSampleBuffer& ioBuffer, MidiBuff
     }
 }
 
-void LpfilterAudioProcessor::process(dsp::ProcessContextReplacing<float> context) noexcept
+void LpfilterAudioProcessor::process (AudioSampleBuffer& processBuffer) noexcept
 {
+    // Define the block that passes into process function
+    dsp::AudioBlock<float> block (processBuffer);
+    
     // lpfJuce filter processing
-    lpfJuce.process(context);
+   // lpfJuce.process(dsp::ProcessContextReplacing<float> (block));
+    
+    if (getTotalNumInputChannels() < 2 )
```

Comment:
I think it would be safer to check the channels of the `processBuffer`. See `AudioBuffer::getNumChannels()`.